fl_training.py

```diff
@@ -375,24 +368,29 @@ def compute_test_score(
     if is_save_fig:
 
         # Save data
-        np.save(save_folder / "score_matrix", score_matrix)
-        np.save(save_folder / "global_val_acc", global_val_acc)
-        np.save(save_folder / "global_val_loss", global_val_loss)
-        np.save(save_folder / "score_matrix_extended", score_matrix_extended)
-
+        metrix_results = {}
```

Comment:
I have 2 questions on that part:

1. I like the sound of "metrix", it is quite cool :-) But still, I think we should write "metrics"?

1. What exactly does the following bit and how can `metrix_results.p` be used for exploring results?
```python
with open(save_folder / "metrix_results.p", 'wb') as f:
            pickle.dump(metrix_results, f)
```